tether/tether.d.ts

```diff
@@ -0,0 +1,50 @@
+// Type definitions for Tether v0.6
+// Project: http://github.hubspot.com/tether/
+// Definitions by: Adi Dahiya <https://github.com/adidahiya>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+declare module tether {
+
+    interface TetherStatic {
+        new(options: ITetherOptions): Tether;
+    }
+
+    interface ITetherOptions {
+        attachment?: string;
+        classes?: {[className: string]: boolean};
+        classPrefix?: string;
+        constraints?: ITetherConstraint[];
+        element?: Element | string | any /* JQuery */;
```

Comment:
this library doesn't depend on `JQuery`... but it accepts `JQuery` objects as an option in some cases... so I made this type `any` and left a comment